.travis.yml

```diff
@@ -2,55 +2,31 @@ language: ruby
 
 rvm:
   - 1.8.7
-  - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1
+  - 2.1.3
```

Comment:
That should be 2.1 plain I think as that will always pick the latest 2.1 which is what we want.